hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,373 @@
+"""
+Unit tests for functions in hibike_message.
+"""
+
+
+import unittest
+import random
+import queue
+import time
+import threading
+
+import serial
+
+import hibike_message
+import spawn_virtual_devices
+from unit_tests.utils import run_with_random_data
+DEVICE_TYPES = list(hibike_message.DEVICES)
+
+
+def random_uid():
+    """
+    Generate a random UID.
```

Comment:
Pep style guide says single line should have quotes on same line
[here](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)